docs/source/user_guide/faq.rst

```diff
@@ -30,6 +30,11 @@ Should I use PySpark's DataFrame API or Koalas?
 If you are already familiar with pandas and want to leverage Spark for big data, we recommend
 using Koalas. If you are learning Spark from ground up, we recommend you start with PySpark's API.
 
+Does Koalas support Structured Streaming?
+-----------------------------------------
+
+No, Koalas does not support Structured Streaming.
```

Comment:
Maybe we can mention `foreachBatch` workaround as discussed offcline.

```python
>>> def func(batch_df, batch_id):
...     koalas_df = ks.DataFrame(batch_df)
...     # Koalas APIs
...     koalas_df['a'] = 1
...     koalas_df.to_csv(...)
...
>>> writer = sdf.writeStream.foreachBatch(func)
```

Let me quickly test it out.

Reply:
Here's working example:

```python
def func(batch_df, batch_id):
    koalas_df = ks.DataFrame(batch_df)
    koalas_df['a'] = 1
    print(koalas_df)

spark.readStream.format("rate").load().writeStream.foreachBatch(func).start()
```

```
                timestamp  value  a
0 2020-02-21 09:49:37.574      4  1
                timestamp  value  a
0 2020-02-21 09:49:38.574      5  1
...
```